src/filters.cpp

```diff
@@ -41,91 +31,73 @@ void Filters::Initialize(v8::Local<v8::Object> target) {
  * // call the function to create filters
  * var filters = new shaver.Filters(styleFilters);
  */
-NAN_METHOD(Filters::New) {
+
+Filters::Filters(Napi::CallbackInfo const& info)
+    : Napi::ObjectWrap<Filters>(info) {
     if (!info.IsConstructCall()) {
-        return Nan::ThrowTypeError(
-            "Cannot call constructor as function, you need to use 'new' keyword");
+        Napi::TypeError::New(Env(), "Cannot call constructor as function, you need to use 'new' keyword");
```

Comment:
Turn `Env()` into a variable at the top and re-use like other code?

Reply:
396786f48f9ec5ea74f6d04aa5a0bc50a3d226f9